packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -1151,23 +1153,37 @@ The following diagram shows the state object for a Device:
+| `displayName`            | String     | The display name of the device (from 1 through 255 characters)                                                    | All                         |
+| `platform`               | Enum       | OS platform of the device. Possible values: `MACOS`, `WINDOWS`, `ANDROID`, `IOS`.                                 | All                         |
+| `registered`             | Boolean    | Indicates if the device is registered at Okta                                                                     | All                         |
+| `diskEncryptionType`     | Enum       | (Optional) The type of disk encryption on the device. [Possible Values for `diskEncryptionType`](#possible-values-for-diskencryptiontype)      | All                         |
+| `imei`                   | String     | (Optional) International Mobile Equipment Identity of the device (from 15 through 17 numeric characters)          | All                         |
+| `integrityJailbreak`     | Boolean    | (Optional) Indicates if the device is jailbroken or rooted                                                        | `IOS` and `ANDROID`         |
+| `manufacturer`           | String     | (Optional) Name of the manufacturer of the device (from 0 through 127 characters)                                 | All                         |
+| `meid`                   | String     | (Optional) Mobile equipment identifier of the device (14 characters)                                              | All                         |
+| `model`                  | String     | (Optional) Model of the device (127 characters)                                                                   | All                         |
+| `osVersion`              | String     | (Optional) Version of the device OS (127 characters)                                                              | All                         |
+| `serialNumber`           | String     | (Optional) Serial number of the device (127 characters)                                                           | All                         |
+| `sid`                    | String     | (Optional) Windows Security identifier of the device (256 characters)                                             | All                         |
+| `udid`                   | String     | (Optional) macOS Unique Device identifier (47 characters)                                                         | All                         |
+| `tpmPublicKeyHash`       | String     | (Optional) Windows Trusted Platform Module hash value                                                             | All                         |
+| `secureHardwarePresent`  | Boolean    | (Optional) Indicates if the device contains a secure hardware functionality                                       | All                         |
+
+##### Possible Values for `diskEncryptionType`
```

Comment:
nit: Possible values for `diskEncryptionType` (we usually prefer sentence-case vs. title-case)

Although I'd probably wan't to get dev docs review to see if this is the right wording cc: @brentschaus-okta